lib/src/chart/radar_chart/radar_chart_painter.dart

```diff
@@ -0,0 +1,310 @@
+import 'dart:developer';
+import 'dart:math' show pi, cos, sin, min;
+import 'dart:ui';
+
+import 'package:fl_chart/src/chart/base/base_chart/base_chart_painter.dart';
+import 'package:fl_chart/src/chart/radar_chart/radar_chart_data.dart';
+import 'package:flutter/material.dart';
+
+import '../../../fl_chart.dart';
+
+class RadarChartPainter extends BaseChartPainter<RadarChartData>
+    with TouchHandler<RadarTouchResponse> {
+  final Paint _borderPaint, _backgroundPaint, _gridPaint, _tickPaint;
+  final Paint _graphPaint, _graphBorderPaint, _graphPointPaint;
+  final TextPainter _ticksTextPaint, _titleTextPaint;
+
+  List<RadarDataSetsPosition> dataSetsPosition;
+
+  RadarChartPainter(
+    RadarChartData data,
+    RadarChartData targetData,
+    Function(TouchHandler) touchHandler, {
+    double textScale = 1,
+  })  : _backgroundPaint = Paint()
+          ..color = data.fillColor
+          ..style = PaintingStyle.fill
+          ..isAntiAlias = true,
+        _borderPaint = Paint()
+          ..color = data?.chartBorderData?.color ?? Colors.black
+          ..strokeWidth = data?.chartBorderData?.width ?? 2
+          ..style = PaintingStyle.stroke
+          ..isAntiAlias = true,
+        _gridPaint = Paint()
+          ..color = data?.gridBorderData?.color ?? Colors.black
+          ..strokeWidth = data?.gridBorderData?.width ?? 2
+          ..style = PaintingStyle.stroke
+          ..isAntiAlias = true,
```

Comment:
We don't need `AntiAlias` in our _gridPaint. If you think it is unnecessary, remove it to achieve more performance.